README.md

```diff
@@ -1,98 +1,146 @@
 # Puppet Server
 
-Puppet Server is the next-generation application for managing Puppet agents.
-This platform will carry Puppet's server-side components to a more
-distributed, service-oriented architecture. We've built Puppet Server on top of the
-same technologies that have made PuppetDB successful, allowing us to
-greatly improve performance, scalability, advanced metrics collection,
-and fine-grained control over the Ruby runtime.
-
-While Puppet Server is meant to replace an existing
-Apache/Passenger Puppet master stack, there are a handful of differences
-due to changes in the underlying architecture. Please see [Puppet Server vs. Apache/Passenger Puppet Master](./documentation/puppetserver_vs_passenger.markdown) for details.
+[Puppet Server](./documentation/services_master_puppetserver.markdown) is the
+next-generation application for managing
+[Puppet](https://github.com/puppetlabs/puppet) agents. This platform implements
```

Comment:
Don't know whether this is a pattern we follow elsewhere, but it seems like maybe it would be more helpful for users reading the documentation to have this link to a docs page on Puppet (or PuppetDB for the link down below) rather than github?